src/client/components/AddResume/AddResume.css

```diff
@@ -48,16 +48,22 @@ h3{
 }
 .upload input{
     width: 85%;
-    height:40px;
+    height:35px;
```

Comment:
Can't we use the button components default styling? 

i.e this: https://github.com/HackYourFuture-CPH/rate-my-cv/tree/develop/src/client/components/Button

Reply:
here i cant use the component styling because the height and width of the button should match the input field.

Reply:
Ah yes, you are right.